### PR TITLE
Brain imaging generator

### PR DESCRIPTION
**Pull request by olivier-bernard-creatis**
Celia, this is your turn to check / correct this post ;)

### CODE REVIEW

File: collections/_posts/2024-01-30-brain-imaging-generation.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+* The generated synthetic dataset is available at [Academics Torrents](https://academictorrents.com/details/63aeb864bbe2115ded0aa0d7d36334c026f0660b) 
+* An inspired code is available on the following [Monai link](https://monai.io/model-zoo.html)
+
+&nbsp;
+
+# Highlights
+
+* Application of the Latent Diffusion Model framework for MR image synthesis of the human brain
+* An encoder/decoder model dedicated to brain MRI reconstruction is proposed
+* Inverstigation of the application of conditioning to age, gender, ventricular volumes and brain volumes
```
Comment (Celia-Gjt) [inline collections/_posts/2024-01-30-brain-imaging-generation.md:24]: Typo on investigation

File: collections/_posts/2024-01-30-brain-imaging-generation.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+&nbsp;
+
+# Highlights
+
+* Application of the Latent Diffusion Model framework for MR image synthesis of the human brain
+* An encoder/decoder model dedicated to brain MRI reconstruction is proposed
+* Inverstigation of the application of conditioning to age, gender, ventricular volumes and brain volumes 
+
+* The model achieves new SOTA for brain MR image synthesis
+* A synthetic dataset of 100,000 volumes together with the conditioning information are publicly available
```
Comment (Celia-Gjt) [inline collections/_posts/2024-01-30-brain-imaging-generation.md:27]: I would say : A synthetic dataset of 100,000 volumes, along with the conditioning information, is publicly available

File: collections/_posts/2024-01-30-brain-imaging-generation.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+* The autoencoder is first trained with a combination of L1 loss, perceptual loss, a patch-based adversarial objective and a KL regularization of the latent space
+
+* The encoder maps the brain image to a latent representation with a size of $$20 \times 28 \times 20$$ voxels
+
+* The diffusion model is then trained using $$1000$$ steps for the Markov chain process 
+
+* The model is conditioned according to age, gender, ventricular volume and brain volume
+
+* The conditioning is performed bu combining the concatenation of the conditioning with the input data and the use of cross-attention mechanisms
```
Comment (Celia-Gjt) [inline collections/_posts/2024-01-30-brain-imaging-generation.md:69]: Typo : bu

File: collections/_posts/2024-01-30-brain-imaging-generation.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+&nbsp;
+
+# Introduction
+
+* The objective of the paper is to generate realistic large scale dataset with additional related "low dimensional" information such as age, sex or volumes.
+
+* 31,740 T1w 3D MR images from the UK Biobank datas are used during training 
+
+* One interest of such dataset would to provide enough data to learn to retrieve the age of a patient based on its brain MR image.
```
Comment (Celia-Gjt) [inline collections/_posts/2024-01-30-brain-imaging-generation.md:37]: while giving privacy guarantees ?

File: collections/_posts/2024-01-30-brain-imaging-generation.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+![](/collections/images/latent-DM/cross-attention.jpg)
+
+&nbsp;
+
+
+# Results
+
+* The autoencoder compressed each dimension of the input data by a factor of 8
+* DDIM is used during inference to reduce from $$1000$$ to $$50$$ the number of time steps during sampling
```
Comment (Celia-Gjt) [inline collections/_posts/2024-01-30-brain-imaging-generation.md:79]: I would add that this  reduce the average sampling time from $$142 \pm 1.6$$s to $$7.6 \pm 0.2$$s.

File: collections/_posts/2024-01-30-brain-imaging-generation.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+
+<div style="text-align:center">
+<img src="/collections/images/brain-image-generation/results-conditioning-age.jpg" width=500></div>
+<p style="text-align: center;font-style:italic">Figure 4. Correlation between inputted age and predicted brain age</p>
+
+&nbsp;
+
+## Synthetic dataset
+
+* a synthetic dataset of 100,000 human brain images was generated and made publicly available together with the conditioning information
```
Comment (Celia-Gjt) [inline collections/_posts/2024-01-30-brain-imaging-generation.md:132]: capital letter on the a ? :)

File: collections/_posts/2024-01-30-brain-imaging-generation.md (hunk 1 above)
Comment (olivier-bernard-creatis) [reply collections/_posts/2024-01-30-brain-imaging-generation.md:24]: Done

File: collections/_posts/2024-01-30-brain-imaging-generation.md (hunk 2 above)
Comment (olivier-bernard-creatis) [reply collections/_posts/2024-01-30-brain-imaging-generation.md:27]: Done

File: collections/_posts/2024-01-30-brain-imaging-generation.md (hunk 3 above)
Comment (olivier-bernard-creatis) [reply collections/_posts/2024-01-30-brain-imaging-generation.md:69]: Done

File: collections/_posts/2024-01-30-brain-imaging-generation.md (hunk 4 above)
Comment (olivier-bernard-creatis) [reply collections/_posts/2024-01-30-brain-imaging-generation.md:37]: Indeed ;)

File: collections/_posts/2024-01-30-brain-imaging-generation.md (hunk 5 above)
Comment (olivier-bernard-creatis) [reply collections/_posts/2024-01-30-brain-imaging-generation.md:79]: Great ;)

**Comment by olivier-bernard-creatis** [comment]
Thanks Celia for your feedback. I fixed all the points you mentioned. If this is fine for you, you can validate the merge request ;)

**Pull request merged**